test/test_array_blobs_big.cpp

```diff
@@ -380,13 +380,28 @@ TEST(ArrayBigBlobs_get_at)
     // Read outside data
     get_pos = 0x2000000;
     read = c.get_at(1, get_pos);
-    CHECK(read.is_null());
+    CHECK(read.size() == 0);
+
+    // Try to assign a new small value to a blob holding a big value. This will fail
+    // if the context flag is not reset properly by the clear operation.
+    c.set(1, BinaryData(lazy_fox));
+    get_pos = 0;
+    read = c.get_at(0, get_pos);
```

Comment:
@jedelbo please confirm this, I believe this should be changed from `0` to `1` ?